src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -7819,12 +7819,13 @@
                   - CoupleColorTempToLevel
       attributes:
           Scenes:
-              - FabricSceneInfo
               # Targeting Spring 2024 Matter release
+              - FabricSceneInfo
```

Comment:
These are in the wrong section too, no?  They should be under "Provisional", not "introduced".